### PR TITLE
33 report reconciled version

### PR DESCRIPTION
**Pull request by BradleyMayo**
Looking for a sanity check before I make the changes to the other crd's and controllers

### CODE REVIEW

File: controllers/webspherelibertyapplication_controller.go [hunk 1]
```diff
@@ -530,6 +530,14 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 		reqLogger.V(1).Info(fmt.Sprintf("%s is not supported", prometheusv1.SchemeGroupVersion.String()))
 	}
 
+	instance.Status.Versions.Reconciled = lutils.OperandVersion
```
Comment (leochr) [inline controllers/webspherelibertyapplication_controller.go:533]: Just setting this line should be enough. The `ManageSuccess` call should write out the status. 

There is another ManageSuccess call above (for Knative scenario). The reconciled version should be added prior to it as well.

**Review by leochr — COMMENTED** [review]
@BradleyMayo Thanks for the PR. Added some comments.

**Comment by BradleyMayo** [comment]
@leochr Everything should be in and ready to merge

**Review by leochr — APPROVED** [review]
@BradleyMayo Looks good. Thank you

**Comment by leochr** [comment]
PR for https://github.com/WASdev/websphere-liberty-operator/issues/33

**Pull request merged**